includes/CAPx/Drupal/Mapper/EntityMapper.php

```diff
@@ -263,6 +296,56 @@ public function mapFieldCollections($data) {
   }
 
   /**
+   * Process entity reference fields uniquely.
+   *
+   * Reference fields are a special field and need to be handled differently.
+   * Allow for the ability to look for an item to attach to from the API
+   * that has already been imported.
+   *
+   */
+  public function mapReferences() {
```

Comment:
From what I can tell, `entityreference` is not a dependency of `stanford_capx`. What happens here if `entityreference` is not enabled?